bigbang/archive.py

```diff
@@ -52,7 +52,11 @@ def __init__(self, data, archive_dir="archives", mbox=False):
         elif isinstance(data, str):
             self.data = mailman.load_data(data,archive_dir=archive_dir,mbox=mbox)
          
-        self.data['Date'] = pd.to_datetime(self.data['Date'], utc=True)
+        try:
+            self.data['Date'] = pd.to_datetime(self.data['Date'], utc=True)
+        except pd.tslib.OutOfBoundsDatetime:
+            # otherwise malformed dates will prevent archive initialization
+            self.data['Date'] = pd.np.nan
```

Comment:
I believe this applies the NaN value to the entire column, which we probably don't want. We should be able to use options on `to_datetime` to coerce values, although I'm also finding bugs with that. But for now, I don't think we can merge this.

Reply:
Yes agreed, I think I merged the wrong change to my master branch, I locally was working on a project that didn't need timestamps and merged in my quick fix that just hardcoded the entire data frame column to NaN rather than the one I meant to open a PR for. I'll find my correct fix and update this PR soon, sorry about this and thanks for catching it!